### PR TITLE
added setup.py

### PR DESCRIPTION
**Pull request by devojoyti**
Also description.txt, which will be later used in PyPI for it's description, not now.

### CODE REVIEW

File: description.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This project is an object oriented Python implementation of the computational framework outlined in Dileep George's PhD thesis, How the Brain Might Work: A Hierarchical and Temporal Model for Learning and Recognition
```
Comment (markroxor) [inline description.txt]: This extra file is not needed. Try integrating this in setup.py instead.

File: requirements.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
-nose==1.3.7
-Pillow==2.9.0
-numpy==1.11.1
-jsonpickle==0.9.3
-coverage
+nose>=1.3.7
```
Comment (markroxor) [inline requirements.txt]: revert these changes for now.

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import os
+import sys
+from setuptools import find_packages, setup
+
+with open('requirements.txt') as f:
```
Comment (markroxor) [inline setup.py]: no. use this https://github.com/RaRe-Technologies/gensim/blob/develop/setup.py#L176

File: requirements.txt (hunk 2 above)
Comment (devojoyti) [inline requirements.txt]: May give installation error, especially in coverage, tried it.

File: setup.py (hunk 3 above)
Comment (devojoyti) [inline setup.py]: It was easier route actually, avoided that because then requirement.txt will be useless, also we may change requirements as needed, much convenient in a text file than in setup.

File: description.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+the human brain's computation principles we will be able to create intelligent
+machines much more intelligent than humans with unimaginable capabilities. At
+first it will not have any emotions like you and I. This will be a good thing
+because it will force us to ask if it would be moral to create an super
+intelligent machine with emotions to help us. It may not necessarily be a
+good thing for the human race. For example, look at how humans treat
+[intelligent pigs](http://modernfarmer.com/2014/03/pigheaded-smart-swine/).
+We are more intelligent emotional machines compared to pigs and the future
+super intelligent emotional machines will be much more intelligent compared to
+us.
```
Comment (markroxor) [inline description.md]: Hardcode this in setup.py instead of a separate file.

File: setup.py (hunk 3 above)
Comment (markroxor) [inline setup.py]: We dont need requirements.txt after that. Keep all the dependencies in setup.py.

File: setup.py [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    description = d.read()
+
+
+setup(
+    name='wAlnut',
+    version='0.0.1',
+    author='Mohit Rathore',
+    author_email='mrmohitrathoremr@gmail.com',
+    maintainer='Devojoyti Halder',
+    maintainer_email='dev.devojoyti@gmail.com',
```
Comment (markroxor) [inline setup.py]: remove this.

File: requirements.txt (hunk 2 above)
Comment (markroxor) [inline requirements.txt]: okay
